<?xml version="1.0" ?>
<testsuite errors="1" failures="1" name="tuqquery.n1ql_fts_integration_phase2_clusterops" skip="0" tests="1" time="0.129314184189">
	<testcase name="tuqquery.n1ql_fts_integration_phase2_clusterops.test_cluster_replicas_failover_rebalance,cluster_name:test,doc-per-day:1,GROUP:P1,spec:test,reset_services:True,num_nodes:4,num_replicas:1,services_init:kv:index:n1ql-kv:index-n1ql:fts-fts,conf_file:test.conf,case_number:1,get-logs:False,stop-on-failure:False,scan_consistency:NOT_BOUNDED,logs_folder:/Users/girishbenakappa/IdeaProjects/testrunner/logs/testrunner-19-Aug-14_11-20-22/test_1,bucket_size:200,nodes_init:4,ini:test.ini" time="0.129314184189">
		<error type="membase.error">[&quot;'module' object has no attribute 'test_cluster_replicas_failover_rebalance'&quot;]</error>
	</testcase>
</testsuite>
